charts/capi-runtime-extensions/templates/csi/aws-ebs/manifests/aws-ebs-csi-configmap.yaml

```diff
@@ -1630,7 +1630,7 @@ data:
                   key: endpoint
                   name: aws-meta
                   optional: true
-            image: public.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver:v1.23.1
+            image: public.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver:v1.23.2
```

Comment:
Thanks for fixing this. lets rebase the dependabot PRs again once you merge this.